src/context.ts

```diff
@@ -32,12 +48,15 @@ export interface ContextValue {
   zapper?: string;
   etherscan?: string;
   addresses?: PartialDeep<AddressesOverride>;
+  simulation?: SimulationConfiguration;
 }
 
 const DefaultContext = {
   // Public API key provided by zapper.
   // see https://docs.zapper.fi/zapper-api/endpoints
-  zapper: "96e0cc51-a62e-42ca-acee-910ea7d2a241"
+  zapper: "96e0cc51-a62e-42ca-acee-910ea7d2a241",
+  // The default tenderly dashboard for Yearn
+  simulation: { dashboardUrl: "https://dashboard.tenderly.co/yearn/yearn-web" }
```

Comment:
@nymmrx added this as the default but also happy to remove if you think it's unnecessary

Reply:
Perfect!